### PR TITLE
Fixes in Kotlin docs

### PR DESCRIPTION
**Pull request by sleipnir**

### CODE REVIEW

File: docs/src/main/paradox/user/lang/kotlin/serialization.md [hunk 1]
```diff
@@ -2,4 +2,8 @@
 
 Cloudstate functions serve gRPC interfaces, and naturally the input messages and output messages are protobuf messages that get serialized to the protobuf wire format. However, in addition to these messages, there are a number of places where Cloudstate needs to serialize other objects, for persistence and replication.
 
-See [here](/docs/src/main/paradox/user/lang/java/serialization.md) for more details
\ No newline at end of file
+<<<<<<< HEAD
```
Comment (marcellanz) [inline docs/src/main/paradox/user/lang/kotlin/serialization.md]: merge conflict to solve.

Comment (sleipnir) [reply docs/src/main/paradox/user/lang/kotlin/serialization.md]: Sorry for this. Send commit to fixed it

**Comment by marcellanz** [comment]
@sleipnir looks good from my side and regarding the issues i found. Thank you.

**Comment by sleipnir** [comment]
> @sleipnir looks good from my side and regarding the issues i found. Thank you.

Thanks @marcellanz for your time and comments

**Comment by sleipnir** [comment]
@viktorklang Ok for merge?

**Comment by viktorklang** [comment]
Thanks @sleipnir! :+1:

**Pull request merged**